spec/features/viewing_posts_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'pg'
+
+feature 'Welcome page' do
+  scenario 'user visits homepage ' do
+    visit('/')
+    expect(page).to have_content "Chitter App"
+    expect(page).to have_link(href: "/posts")
+    expect(page).to have_link(href: "/posts/new")
+  end
+end
+
+feature 'Viewing posts' do
+  scenario 'visiting /posts shows message' do
+    
+    Post.create(date: "2022-04-01", author: "Kate", message: "How are you?")
+
+    visit('/posts')
+    expect(page).to have_content("2022-04-01")
+    expect(page).to have_content("Kate")
+    expect(page).to have_content("How are you?")
+
+  end
+end
+
+feature 'Viewing posts' do
+  scenario 'visiting /posts shows new messages first' do
+    
+    Post.create(date: "2022-04-01", author: "Kate", message: "How are you?")
+    Post.create(date: "2022-06-01", author: "Kate", message: "How are you?")
+    Post.create(date: "2022-08-01", author: "Kate", message: "How are you?")
+
+    visit('/posts')
+    expect(page).to have_content("2022-08-01")
+    expect(page).to have_content("2022-06-01")
+    expect(page).to have_content("2022-04-01")
```

Comment:
This doesn't actually check the order of the content. You could use regular expressions here to check that "2022-08-01" is above "2022-06-01", for example with something like
```rb
expect(page).to have_content("2022-08-01.*2022-06-01")
```

However, I think an even better test would check the content of the mew message comes before the content of an older one, this way the test would still pass if you decide to change the formatting of the date, given that the user story is about posts order, not dates :)